### PR TITLE
Shift on Stack DT

### PR DESCRIPTION
**Pull request by mandre**
Depends on https://github.com/openstack-k8s-operators/architecture/pull/244

Renamed the Shift on Stack overrides file from `shift-on-stack-overrides.yml` to `dt-osasinfra.yml` to highlight that it now deploys the osasinfra DT.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by mandre** [comment]
The [dependent PR](https://github.com/openstack-k8s-operators/architecture/pull/244) merged. @cjeanner could you review when you get a chance?

**Comment by cjeanner** [comment]
recheck

**Comment by mandre** [comment]
@abays you recently reviewed the osasinfra DT patch (https://github.com/openstack-k8s-operators/architecture/pull/244), this is the needed cifm patch to enable it. Would you be able to take a look?

**Comment by mandre** [comment]
> @abays you recently reviewed the osasinfra DT patch ([openstack-k8s-operators/architecture#244](https://github.com/openstack-k8s-operators/architecture/pull/244)), this is the needed cifm patch to enable it. Would you be able to take a look?

@fultonj as well :wink:

File: scenarios/reproducers/dt-osasinfra.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      disksize: 50
+      memory: 8
+      cpus: 4
+      nets:
+        - ocpbm
+        - osp_trunk
+
+## devscript support for OCP deploy
+cifmw_devscripts_config_overrides:
+  fips_mode: "{{ cifmw_fips_enabled | default(false) | bool }}"
```
Comment (abays) [inline scenarios/reproducers/dt-osasinfra.yml:128]: I know certain things were moved out of `cifmw_devscripts_config_overrides` during #1777.  Is `fips_mode` still valid here?

Comment (mandre) [reply scenarios/reproducers/dt-osasinfra.yml:128]: Not sure if it's needed, or has any effect. I copied [from the va-hci.yml reproducer](https://github.com/openstack-k8s-operators/ci-framework/blob/917db7b/scenarios/reproducers/va-hci.yml#L111-L113).

Comment (abays) [reply scenarios/reproducers/dt-osasinfra.yml:128]: @cjeanner Do you know if `fips_mode` is still a valid option in `cifmw_devscripts_config_overrides`?

Comment (cjeanner) [reply scenarios/reproducers/dt-osasinfra.yml:128]: it is, yes

**Review by fultonj — COMMENTED** [review]
+2

**Comment by rlobillo** [comment]
Tested on https://sf.hosted.upshift.rdu2.redhat.com/zuul/t/tripleo-ci-internal/build/2c5369b3e12348998b62c8661ba85be6

**Pull request merged**